org/invite-collaborator.ts

```diff
@@ -52,30 +55,37 @@ export const inviteCollaborator = wrap(
 `;
 
     try {
-      const invite = await api.orgs.addTeamMembership({
-        // ID of the @gatsbyjs/maintainers team on GitHub
-        id: '1942254',
-        username
-      });
-
-      if (invite.data.state === 'active') {
-        console.log(
-          `@${username} is already a ${invite.data.role} for this team.`
-        );
-      } else {
-        console.log(`We’ve invited @${username} to join this team.`);
+      // This throws if the user isn't a member of the org yet.
+      // If it doesn't throw, it means the user was already invited or has
+      // already accepted the invitation (we ignore the return value here).
+      await api.orgs.checkMembership({ org, username });
+    } catch (error) {
+      // This is a query to check the number of merged pull requests by the user.
+      const query = `org:gatsbyjs+author:${username}+type:pr+is:merged`;
+      const response = await api.search.issues({ q: query });
+      const mergedCount = response.data.total_count;
+      // We only send out an invite on the first PR, in order not to spam users.
+      if (mergedCount === 1) {
+        try {
+          const invite = await api.orgs.addTeamMembership({
+            // ID of the @gatsbyjs/maintainers team on GitHub
+            id: '1942254',
+            username
+          });
+    
+          if (invite.data.state === 'active') {
+            console.log(
+              `@${username} is already a ${invite.data.role} for this team.`
+            );
+          } else {
+            console.log(`We’ve invited @${username} to join this team.`);
+          }
+        } catch (err) {
+          console.log('Something went wrong.');
+          console.log(err);
+        }
+        markdown(comment)
```

Comment:
We shouldn't need to use create comment. At Moya we are using the `markdown` function for this

Reply:
I tried this earlier and it didn't work, but that may have been something else I was doing wrong.